src/components/table-transactions/table-transactions.js

```diff
@@ -0,0 +1,40 @@
+import angular from "angular";
+import uiGrid from "angular-ui-grid";
+import angularAnimate from "angular-animate";
```

Comment:
when you add alias you should use this alias.
otherwise you don't need the alias